packages/web-components/src/components/image-with-caption/__stories__/image-with-caption.stories.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import { boolean } from '@storybook/addon-knobs';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import '../image-with-caption';
+import readme from './README.stories.mdx';
+
+export const Default = ({ parameters }) => {
+  const { alt, defaultSrc, heading, copy, lightbox } = parameters?.props?.['dds-image-with-caption'] ?? {};
+  return html`
+    <dds-image-with-caption
+      alt="${alt}"
+      default-src="${defaultSrc}"
+      heading="${heading}"
+      copy="${copy}"
+      ?lightbox="${lightbox}"
+    />
```

Comment:
```suggestion
    ></dds-image-with-caption>
```

Reply:
@asudoh no worries! sorry I missed it earlier. Just updated again